amplfi/train/data/datasets/base.py

```diff
@@ -545,7 +544,10 @@ def inject(self, *args, **kwargs):
         raise NotImplementedError
 
     def background_from_gpstimes(
-        self, gpstimes: np.ndarray, fnames: List[Path]
+        self,
+        gpstimes: np.ndarray,
+        fnames: List[Path],
+        timeslides: Optional[np.ndarray] = None,
```

Comment:
should we raise a RunTimeError checking that the length of `gpstimes` and `timeslides` is the same.

Reply:
good idea